main.py: Running...
Evaluating .p_do_solving...
Got an exception: raised LIBFOOLANG.ANALYSIS.PROPERTY_ERROR : invalid equation for logic resolution

main.py: Done.
Done
